### ISSUE TITLE
No repository visible as '@arm_none_linux_gnueabihf' from main repository

### ISSUE THREAD

**Issue by makestuffwork** [hexdae/bazel-arm-none-eabi#46]
I'm getting this error:
"While resolving toolchains for target : invalid registered toolchain '@arm_none_linux_gnueabihf//toolchain:all': No repository visible as '@arm_none_linux_gnueabihf' from main repository"

What am I missing? Thanks!

**Comment by makestuffwork** [comment]
What's missing is this line under https://github.com/hexdae/bazel-arm-none-eabi?tab=readme-ov-file#bzlmod
```
arm_toolchain.arm_none_linux_gnueabihf()
use_repo(arm_toolchain, **"arm_none_linux_gnueabihf"**)
register_toolchains("@arm_none_linux_gnueabihf//toolchain:all")
```

**Comment by hexdae** [comment]
Hey @makestuffwork, thanks a lot for finding this issue, here's a PR to fix it #47

**Comment by hexdae** [comment]
The PR should address this, if not, let me know and I will re-open
